packages/experimental/Button/src/Button.tsx

```diff
@@ -47,12 +47,13 @@ export const Button = compose<ButtonType>({
 
     // now return the handler for finishing render
     return (final: ButtonProps, ...children: React.ReactNode[]) => {
-      const { icon, iconPosition, content, ...mergedProps } = mergeProps(button.props, final);
```

Comment:
We need to be careful about breaking changes.  Do we really want to drop support for content entirely?  If so, can we stage it to avoid breaking folks downstream?

Reply:
We can add deprecation notice. Or we can get the list of those who's already using it and if there are a couple of users we can ask them to update according to the latest version.

Reply:
We have been changing/removing props from the experimental button a decent amount for the last few weeks. Since this package isn't shipped with the main @fluentui/react-native bundle, we've been kind of considering it as not shipped and I haven't heard any feedback about the various breaking changes I've been making to this button over the weeks.

I've written up a Migration document which specifies how to move from the old button to the new as a guide for clients, and we will mark the old button as deprecated when we're ready to move this button into the main Button package.

Reply:
Ok, sounds good to me!